trino/trino.go

```diff
@@ -1043,6 +1114,40 @@ func (qr *driverRows) initColumns(qresp *queryResponse) error {
 	return nil
 }
 
+func (qr *driverRows) updateCallback(id string, stats stmtStats) {
+	if qr.stmt.conn.callback == nil {
+		return
+	}
+
+	// Check and return if statsCh Channel is closed
+	select {
+	case <-qr.statsCh:
```

Comment:
don't you read and drop value from `statsCh` with this statement if this is not closed?

Reply:
Even if this code is removed, thx for pointint it out.
- https://github.com/trinodb/trino-go-client/pull/14#discussion_r949984630

I tested it at [playground](https://go.dev/play/p/5arKW_2EQJ3) and very interestingly `poing` disappears, however, it appears if I uncomment line 34 and send another message `messages <- "new pong"`
```
// before uncomment
ping
pong
before open
where is my pong?

...
//after uncomment
ping
before open
where is my pong?
pong
new pong